static/app/components/assigneeSelector.tsx

```diff
@@ -178,8 +179,8 @@ const StyledIconUser = styled(IconUser)`
   margin-right: 2px;
 `;
 
-const StyledChevron = styled(IconChevron)`
-  margin-left: ${space(1)};
+const StyledChevron = styled(Chevron)`
+  margin-left: ${space(0.5)};
```

Comment:
Can we use gap instead on the `DropdownButton`

Reply:
👍